<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_gen_file">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_bg_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj1_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj0_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_window_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_bg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BG         :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_obj0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OBJ0    :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_obj1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OBJ1    :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_window">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Window:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scale">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>darkest &lt;-----------------------------------------------------------------------------------------&gt; brightest</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hexdecimal Color Values</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj1_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj0_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_bg_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_window_1">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj1_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj0_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_bg_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_window_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj1_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_obj0_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_bg_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_window_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bg_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj0_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj1_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_window_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_window_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj0_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bg_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj1_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_window_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj0_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bg_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj1_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_window_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj0_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bg_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_obj1_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAnalogue_Pocket_Gameboy_Palettes">
    <property name="title">
     <string>Analogue Pocket Gameboy Palettes</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuAnalogue_Pocket_Gameboy_Palettes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
